docs/host/host.md

```diff
@@ -284,4 +284,140 @@ $ kubectl get nodes harvester-node-0 -o yaml |yq -e '.metadata.annotations.["nod
+The YAML document in the .spec.contents field will be rendered to `/oem/99_ssh.yaml` (due to the .spec.filename field.)
+
+Apply this example with `kubectl apply -f ssh_access.yaml`.
+
+:::tip
+Don't forget to reboot the relevant Harvester nodes so that the Elemental toolkit executor can apply the new configuration at boot.
+:::
+
+#### Understanding the CloudInit resource spec
+
+| Field | Required | Description |
+| - | - | - |
+| matchSelector | Yes | Allows customizing which nodes receive the configuration change. |
+| filename | Yes | The filename that appears under `/oem`. |
+| contents | Yes | An Elemental toolkit cloud-init-style file; it will be rendered to a file under `/oem`. |
+| paused | No | If `true`, the file will not be updated on nodes as it changes. |
+
+The `matchSelector` field can be used to target specific nodes or groups of nodes based on their labels.
+
+For example, to target a specific node, the matchSelector could look like this:
+
+```yaml
+matchSelector:
+  kubernetes.io/hostname: "harvester-node-1"
+```
+
+:::note
+All of the label key-value pairs listed in the `matchSelector` field must match the intended nodes' labels.
+For example, the following `matchSelector` would not match `harvester-node-1` if that node didn't have the `example.com/role` label with the `role-a` value, even though the `kubernetes.io/hostname` label _does_ match.
+
+```yaml
+matchSelector:
+  kubernetes.io/hostname: "harvester-node-1"
+  example.com/role: "role-a"
+```
+:::
+
+### Updating a CloudInit resource
+
+`kubectl edit` is sufficient; however, there is a caveat if the `matchSelector` field is updated to exclude one or more nodes from the customization.
+Please review the note in the [Deleting a CloudInit resource section](#deleting-a-cloudinit-resource) regarding rolling back customizations.
+
+```console
+# kubectl edit cloudinit CLOUDINIT_NAME
+```
+
+### Deleting a CloudInit resource
+
+To remove a CloudInit resource from the Harvester cluster, then `kubectl delete` is sufficient:
+
+```console
+# kubectl delete cloudinit CLOUDINIT_NAME
+```
+
+:::note
+Harvester is not able to "roll back" any customizations that the CloudInit resource previously described, since the CloudInit resource can describe anything that can be expressed as an Elemental toolkit customization, including arbitrary shell commands.
+
+For instance, in the [Creating a CloudInit Resource section](#creating-a-cloudinit-resource), the example uses the Elemental toolkit `authorized_keys` stanza.
+This is an append-only action in the Elemental toolkit, so when that resource is changed or deleted, the Rancher's authorized_keys file will still contain the old public key.
+
+**It is the cluster operator's responsibility to amend or create a CloudInit resource that rolls the changes back if necessary prior to rebooting the node.**
+:::
```

Comment:
```suggestion
:::note
Harvester is unable to "roll back" previously described customizations because the CloudInit resource can describe anything that can be expressed as an Elemental toolkit customization, including arbitrary shell commands.

In the [Creating a CloudInit Resource](#creating-a-cloudinit-resource) example, the YAML file contains the `authorized_keys` stanza. This is an append-only action in the Elemental toolkit. When the resource is changed or deleted, the `authorized_keys` file in Rancher will still contain the old public key.

**You are responsible for amending or creating a CloudInit resource that rolls the changes back (if necessary) before you reboot the node.**
:::
```